compressors/rANS.py

```diff
@@ -84,31 +87,28 @@ class rANSParams:
 
     # rANS state is limited to the range [RANGE_FACTOR*total_freq, (2**NUM_BITS_OUT)*RANGE_FACTOR*total_freq - 1)]
     # RANGE_FACTOR is a base parameter controlling this range
-    RANGE_FACTOR_BITS: int = 16
-    RANGE_FACTOR: int = 1 << RANGE_FACTOR_BITS
-
-    def initial_state(self, freqs: Frequencies) -> int:
-        """the initial state from which rANS encoding begins
+    RANGE_FACTOR: int = 1 << 16
```

Comment:
The write-up doesn't mention RANGE_FACTOR. Maybe add here that RANGE_FACTOR is same as M in write-up, or if this is a more commonly used term, we should add it in write-up.

Reply:
also, I thought you would choose NUM_BITS_OUT to be 8 or 16 for speed reasons. Is it a reason it is 1 here. Should we test that the code works with higher NUM_BITS_OUT as well and leads to some speedup depending on architecture?

Reply:
@tpulkit  -> resolved by mentioning RANGE_FACTOR in the writeup